dandi/dandiapi.py

```diff
@@ -1,3 +1,31 @@
+"""
+Example code for downloading all assets from all published Dandisets that have
+"two-photon" in their ``metadata.measurementTechnique[].name``:
+
+.. code:: python
+
+    from operator import attrgetter
+    from pathlib import Path
+    from dandi.dandiapi import DandiAPIClient
+
+    with DandiAPIClient.for_dandi_instance("dandi") as client:
+        for dandiset in client.get_dandisets():
+            published_versions = [
+                v for v in dandiset.get_versions() if v.identifier != "draft"
+            ]
+            if not published_versions:
+                continue
+            latest_version = max(published_versions, key=attrgetter("created"))
```

Comment:
don't we expose somehow the `most_recent_published_version` API provides for the dandiset record?

Reply:
Yes; forgot about it.